test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SignatureUtilityTests.cs

```diff
@@ -80,25 +80,25 @@ public void GetPrimarySignatureCertificates_WithAuthorSignature_ReturnsCertifica
         public void GetPrimarySignatureTimestampSignatureCertificates_WhenSignatureNull_Throws()
         {
             var exception = Assert.Throws<ArgumentNullException>(
-                () => SignatureUtility.GetPrimarySignatureTimestampSignatureCertificates(signature: null));
+                () => SignatureUtility.GetPrimarySignatureTimestampCertificates(signature: null));
```

Comment:
This test (and others) were testing the method `GetPrimarySignatureTimestampSignatureCertificates(...)`.  The method name begins with the name of the member under test.  Since you renamed the name of the member under test, update the test name too.  This feedback applies globally.

    <MemberName>_<TestCondition>_<ExpectedResult>